src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -209,6 +212,8 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
       </div>
     </fieldset>
   </form>
-  <alm-work-item-link [workItem]="workItem" [loggedIn]="loggedIn"></alm-work-item-link>
-  <alm-work-item-comment [workItem]="workItem" [loggedIn]="loggedIn"></alm-work-item-comment>
-</div>
+  <div *ngIf="!addNewWI">
```

Comment:
Well, this actually means we can't create a link or comment until the item is created. I guess this is fine.